Source/VMware.vSphereDSC/Configurations/ESXiConfigs/VMHostVss/VMHostVssBridge_Config.ps1

```diff
@@ -0,0 +1,75 @@
+<#
```

Comment:
Rebase with master because now configurations are present in a new folder.

Reply:
Is that with this command?
```
git rebase upstream/master
```

Reply:
Yes, this should work. And after that move your configuration file to the right folder.

Reply:
Updated